canonical/ght
# Added sentry configuration

## Done
- Sentry configuration is added

## QA
- Checkout the feature branch
- Install dependencies with `yarn`
- Add `throw new Error("Error...")` to somewhere in the code that will be executed when the 'replicate' command is executed
- Run `yarn dev replicate -i`
- Verify that no error is displayed in the [Sentry](https://sentry.is.canonical.com/canonical/ght/) 
- Change `"dev": "NODE_ENV=development ts-node ./src/index.ts"` to `"dev": "NODE_ENV=production ts-node ./src/index.ts"` in the package.json
- Run `yarn dev replicate -i`
- Verify that the error is displayed in the [Sentry](https://sentry.is.canonical.com/canonical/ght/) 

Note: Be aware of when the `dev` command is changed, the job posts will be created to the "Canonical-Jobs" board.

Fixes #113
